packages/design-system/src/themes/default/index.css

```diff
@@ -29,4 +29,14 @@
   --ads-highlight-text-default-text-color: var(--ads-color-orange-800);
   --ads-highlight-text-default-background-color: var(--ads-color-orange-100);
 
+  /* Tag Input */
+  --ads-tag-input-background-color: var(--ads-color-black-0);
+  --ads-tag-input-border-color: var(--ads-color-black-250);
+  --ads-tag-input-color: var(--ads-color-black-850);
```

Comment:
Can we make sure to specify what color everywhere? The asumption that "color" by itself is text-color is something I want to avoid